clang/test/SemaHLSL/BuiltIns/cross-errors.hlsl

```diff
@@ -0,0 +1,43 @@
+// RUN: %clang_cc1 -finclude-default-header -triple dxil-pc-shadermodel6.6-library %s -fnative-half-type -disable-llvm-passes -verify -verify-ignore-unexpected

```

Comment:
Is this producing or expected to produce unexpected diagnostics? I see this flag in a few other uses of this in SemaHLSL, but not so much in other Sema* test directories. Perhaps it's getting copied over needlessly? 

If there are some tricky diagnostics getting produced, it might be better to limit them by assigning tis to a specific type of diagnostic.

Reply:
Thanks, this revealed a missing open curly brace on line 18!
I got rid of this extra flag, seems to be a needless copy.